backend/src/test/java/gov/cdc/usds/simplereport/service/TestResultUploadServiceTest.java

```diff
@@ -95,23 +96,24 @@ void ioException_CaughtAndThrowsCsvException() throws IOException {
   @DirtiesContext
   @SliceTestConfiguration.WithSimpleReportCsvUploadPilotUser
   void feignBadRequest_returnsErrorMessage() throws IOException {
-
-    stubFor(
-        WireMock.post(WireMock.urlEqualTo("/api/reports"))
-            .willReturn(
-                WireMock.aResponse()
-                    .withStatus(HttpStatus.BAD_REQUEST)
-                    .withHeader("Content-Type", MediaType.APPLICATION_JSON_VALUE)
-                    .withBody("you messed up")));
+    try (var x = loadCsv("responses/datahub-error-response.json")) {
+      stubFor(
+          WireMock.post(WireMock.urlEqualTo("/api/reports"))
+              .willReturn(
+                  WireMock.aResponse()
+                      .withStatus(HttpStatus.BAD_REQUEST)
+                      .withHeader("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+                      .withBody(x.readAllBytes())));
+    }
     InputStream input = mock(InputStream.class);
     when(input.readAllBytes()).thenReturn(new byte[] {45});
 
     var response = this._service.processResultCSV(input);
 
-    assertEquals(1, response.getErrors().length);
+    assertEquals(6, response.getErrors().length);
     var errorMessage = Arrays.stream(response.getErrors()).findFirst().get();
```

Comment:
Looks like this variable is unused, did you want to run an assertion against it?